data/json/items/ammo.json

```
@@ -1327,5 +1327,26 @@
     "count": 50,
     "ammo_type": "tinder",
     "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "rpg_dice",
+    "price": 10,
+    "name": "rpg dice",
+    "name_plural": "rpg dice",
+    "symbol": "=",
+    "color": "light_gray",
+    "looks_like": "marble",
+    "description": "Some dice used to play various role playing games, coming in several shapes and sizes",
```

The description should be for that of a singular die (rather than "some dice"). Also, "role-playing" is hyphenated

Eg

```suggestion
    "description": "A die often used to play various role-playing games",
```

---

There's no reason the item can't depict a set of dice.

---

Per the PR:

> The dice are (unless I messed up) usable as ammunition for the sling and slingshot. They are pretty much marbles with slightly adjusted stats such as less damage and higher range.

It probably makes more sense to be shooting individual die, no?